### PR TITLE
Add how to engage section

### PR DESCRIPTION
**Pull request by Bestem0r**
## Description

By request from HS, a "Hvordan engasjere seg?" section has been added

## Screenshot

Screenshot from 2024-08-24 15-15-31

### CODE REVIEW

File: pages/om-abakus.tsx [hunk 1]
```diff
@@ -137,7 +137,42 @@ const OmAbakus = () => {
...
+          <Link href="https://opptak.abakus.no">opptak.abakus.no</Link> og vil
+          annonseres på <Link href="https://abakus.no">abakus.no</Link> og i
+          sosiale medier.
+        </p>
+        <br />
+        <b>Undergrupper</b>
+        <p>
+          Undergruppene sine opptak varierer fra gruppe til gruppe. Se hver
+          enkelt undergruppe sin side på{" "}
+          <Link href="https://abakus.no">abakus.no</Link> for mer informasjon.
```
Comment (norbye) [inline pages/om-abakus.tsx:158]:
```suggestion
          <Link href="https://abakus.no/pages/grupper/31-undergrupper">abakus.no</Link> for mer informasjon.
```
https://abakus.no/pages/grupper/31-undergrupper has potential, but at the very least it saves people clicking the link the hassle of having to locate the info themselves (which is not that easy imo)

File: pages/om-abakus.tsx [hunk 2]
```diff
@@ -137,7 +137,42 @@ const OmAbakus = () => {
...
-      <h3 className={styles.subTitle}>Komiteer</h3>
+      <AboutCard title="Hvordan engasjere seg?">
+        <b>Komiteene og revyen</b>
+        <p>
+          Opptak til{" "}
+          <Link href="https://abakus.no/pages/grupper/104-revyen">revyen</Link>{" "}
+          og <Link href="#comittees">komiteene</Link> skjer vanligvis i
+          september etter semesterstart.{" "}
+          <Link href="https://abakus.no/pages/komiteer/5">backup</Link> har
+          vanligvis opptak i februar. \n Opptaket gjennomføres på{" "}
```
Comment (norbye) [inline pages/om-abakus.tsx:148]: Remove the \n

File: pages/om-abakus.tsx [hunk 3]
```diff
@@ -137,7 +137,42 @@ const OmAbakus = () => {
...
+        <p>
+          Undergruppene sine opptak varierer fra gruppe til gruppe. Se hver
+          enkelt undergruppe sin side på{" "}
+          <Link href="https://abakus.no">abakus.no</Link> for mer informasjon.
+        </p>
+        <br />
+        <b>Interessegrupper</b>
+        <p>
+          Interessegruppene i Abakus er åpne for alle. Man kan melde seg inn ved
+          å trykke på {'"'}Bli med i gruppen{'"'} på interessegruppens side på{" "}
```
Comment (norbye) [inline pages/om-abakus.tsx:164]: Is the {'"'} really needed?

**Review by norbye — CHANGES_REQUESTED** [review]

File: pages/om-abakus.tsx (hunk 3 above)
Comment (Bestem0r) [reply pages/om-abakus.tsx:164]: Hmm, I personally think so as it indicates that "Bli med i gruppen" is the button label

Comment (norbye) [reply pages/om-abakus.tsx:167]: Ah yeah I meant if you can't just type it as " in the text

**Review by norbye — APPROVED** [review]

File: pages/om-abakus.tsx (hunk 3 above)
Comment (Bestem0r) [reply pages/om-abakus.tsx]: Ah well the linter didn't really like it, but I found a solution now using `&quot;` instead.

**Pull request merged**